.eslintrc.json

```diff
@@ -90,6 +96,12 @@
       "varsIgnorePattern": "^_",
       "argsIgnorePattern": "^_"
     }],
+    "@typescript-eslint/no-floating-promises": [
+      "error",
+      {
+        "ignoreVoid": true
```

Comment:
Why are you ignoring void? I can imagine some asynchronous functions that are void and still need to be awaited